{"name": "test_vwo_login_negative", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b2e169298cbc45b2b7c3c0284f89f3de\")>\n\n    @allure.epic(\"VWO login\")\n    @pytest.mark.negative\n    def test_vwo_login_negative(setup):\n        driver = setup\n        loginpage = LoginPage(driver)\n        loginpage.login_to_vwo(usr=\"admin@gmail.com\", pwd=\"admin\")\n        time.sleep(20)\n>       error_message = loginpage.get_error_message_text()\n\ntests\\vwoLoginTests\\test_vwo_login.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntests\\pageObjects\\loginPage.py:59: in get_error_message_text\n    return self.get_error_message_text()\ntests\\pageObjects\\loginPage.py:59: in get_error_message_text\n    return self.get_error_message_text()\nE   RecursionError: maximum recursion depth exceeded\n!!! Recursion detected (same locals & position)"}, "start": 1715950000371, "stop": 1715950023618, "uuid": "c4b593d6-51c3-40e7-a554-a5823848f62b", "historyId": "d768d3d769f5652f534d483ce8b8f9ed", "testCaseId": "d768d3d769f5652f534d483ce8b8f9ed", "fullName": "tests.vwoLoginTests.test_vwo_login#test_vwo_login_negative", "labels": [{"name": "epic", "value": "VWO login"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-K766CG4"}, {"name": "thread", "value": "7520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwo_login"}]}